common/api/core-frontend.api.md

```diff
@@ -10654,9 +10658,21 @@ export class SectionDrawingModelState extends DrawingModelState {
     static get className(): string;
 }
 
+// @public
+export interface SelectableIds {

```

Comment:
Why make every property required here if you're going to use `Partial<SelectableIds>` everywhere in the API?

Reply:
Agree, makes no sense. Made the members optional.